learn/advanced/filtering_and_faceted_search.md

```diff
@@ -84,54 +80,178 @@ director = 'Jordan Peele'
 director = "Tim Burton"
 ```
 
-Another condition could request movies released after 18 March 1995 (written as 795484800 in UNIX Epoch time):
+Another condition could request movies released after 18 March 1995 (written as `795484800` in UNIX Epoch time):
 
 ```
 release_date > 795484800
 ```
 
 ::: warning
-As no specific schema is enforced at indexing, the filtering engine will try to coerce the type of `value`. This can lead to undefined behavior when big floats are coerced into integers and reciprocally. For this reason, it is best to have homogeneous typing across fields, especially if numbers tend to become large.
+Since Meilisearch does not enforce a specific schema when indexing data, the filtering engine will try to coerce the type of `value`. This can lead to undefined behavior when big floats are coerced into integers and reciprocally. For this reason, it is best to have homogeneous typing across fields, especially when dealing with large numbers.
+:::
+
+### Filter operators
+
+Meilisearch supports the following filter operators:
+
+#### Equality
+
+The equality operator (`=`) returns all documents containing an attribute equal to a specific value. When operating on strings, `=` is case-insensitive.
```

Comment:
I had to re-read the first sentence a few times. It strikes me as accurate but a tad long/confusing. Any way to simplify it, or break it into multiple sentences? 🤔

Reply:
Reworded to:
The equality operator (`=`) returns all documents containing a specific value for a given attribute.